CODEOWNERS

```diff
@@ -1 +1 @@
-* @puppetlabs/puppetdb @bastelfreak @smortex @h0tw1r3
```

Comment:
@chambersmp why do you remove @h0tw1r3 ?

Reply:
@bastelfreak @h0tw1r3 doesn't appear to have `write` access anymore to be a Codeowner.